api/rest/triple.yaml

```diff
@@ -0,0 +1,109 @@
+---
+swagger: '2.0'
+info:
+  title: Data Commons API
+  description: Given a list of nodes, return triples which are associated with the specified node(s).
```

Comment:
Can we add a short description of triples, or a pointer to one? Put another way, should this page standalone on it's own - if a user were to only see this page, should they know what to do without reading the rest of the site? Some questions might be: what is a triple? what is a dcid? Perhaps this is something we can address in a nicer intro page to the API.